tests/exercise_instructions.rs

```diff
@@ -0,0 +1,506 @@
+
+                test_ins(
+                    format!(
+                        "mov32 r{tmp},{}
+                         mov64 r{src},{:#x}
+                         lsh64 r{src},32
+                         or64 r{src},r{tmp}
+                         ldxh r{dst}, [r{src}{offset:+}]",
+                        addr as i32,
+                        addr >> 32,
+                    ),
+                    &mut prng,
+                    25,
+                );
+
+                test_ins(
+                    format!(
+                        "mov32 r{tmp},{}
+                         mov64 r{src},{:#x}
+                         lsh64 r{src},32
+                         or64 r{src},r{tmp}
+                         stxh [r{src}{offset:+}], r{dst}",
+                        addr as i32,
+                        addr >> 32,
+                    ),
+                    &mut prng,
+                    25,
+                );
+
+                let addr = rand % 77 + 0x4_0000_0000i64 - offset as i64;
+                test_ins(
+                    format!(
+                        "mov32 r{tmp},{}
+                         mov64 r{src},{:#x}
+                         lsh64 r{src},32
+                         or64 r{src},r{tmp}
+                         ldxw r{dst}, [r{src}{offset:+}]",
+                        addr as i32,
+                        addr >> 32,
+                    ),
+                    &mut prng,
+                    25,
+                );
+                test_ins(
+                    format!(
+                        "mov32 r{tmp},{}
+                         mov64 r{src},{:#x}
+                         lsh64 r{src},32
+                         or64 r{src},r{tmp}
+                         stxw [r{src}{offset:+}], r{dst}",
+                        addr as i32,
+                        addr >> 32,
+                    ),
+                    &mut prng,
+                    25,
+                );
+
+                let addr = rand % 73 + 0x4_0000_0000i64 - offset as i64;
+
+                test_ins(
+                    format!(
+                        "mov32 r{tmp},{}
+                         mov64 r{src},{:#x}
+                         lsh64 r{src},32
+                         or64 r{src},r{tmp}
+                         ldxdw r{dst}, [r{src}{offset:+}]",
+                        addr as i32,
+                        addr >> 32,
+                    ),
+                    &mut prng,
+                    25,
+                );
+                test_ins(
+                    format!(
+                        "mov32 r{tmp},{}
+                         mov64 r{src},{:#x}
+                         lsh64 r{src},32
+                         or64 r{src},r{tmp}
+                         stxdw [r{src}{offset:+}], r{dst}",
+                        addr as i32,
+                        addr >> 32,
+                    ),
+                    &mut prng,
+                    25,
+                );
```

Comment:
What about loading and storing immediates? `lddw r2, 0xfffffffffffffffff`? `stdw [dst+off], imm`?

Reply:
stdw is on line 418
lddw is no longer supported

Reply:
> lddw is no longer supported

SBFv2 is not yet enabled. Until we enable it, lddw is still emitted by the compiler.

Reply:
ok I've added an sbfv1 lddw test